web/src/main/resources/static/js/tree.js

```diff
@@ -113,6 +113,10 @@ var tree = (function () {
                             uniqueTreeNode[type].umls = row.metaumls;
                         }
 
+                        if(row.hasOwnProperty('history')){
+                            uniqueTreeNode[type].history = row.history;
```

Comment:
parse new field from tumortypes.txt output .. if we rename the field in the output of tumortypes.txt we should match it here.